src/libaktualizr/uptane/iterator.h

```diff
@@ -0,0 +1,65 @@
+#ifndef AKTUALIZR_UPTANE_ITERATOR_H_
+#define AKTUALIZR_UPTANE_ITERATOR_H_
+
+#include "fetcher.h"
+#include "imagesrepository.h"
+
+namespace Uptane {
+
+Targets getTrustedDelegation(const Role &delegate_role, const Targets &parent_targets,
+                             const ImagesRepository &images_repo, INvStorage &storage, Fetcher &fetcher);
```

Comment:
Can this be static?

Reply:
Never mind, I overlooked that this is outside of a class, so the benefit of `static` that I was hoping for is already there.